src/styles/blocks/main1.scss

```diff
@@ -0,0 +1,24 @@
+
+.main {
+  width: 1024px;
```

Comment:
don't hardcode width, your catalog container should work fine for any number of cards